### DISCUSSION TITLE
How to limit myself on a page after I already accessed it?

### DISCUSSION THREAD

**Discussion by kitten7280** [proginosko/LeechBlockNG#405] [Q&A] [answered]
Let's take an example of Twitter main feed page (twitter.com/home). 

When I first open the page, I want to be redirected to a delay. After I waited for the period of time I'm now at Twitter feed. But in this tab after the delay I don't have any obstacles, I can freely spend an hour in it and it's bad. Some ideas for possible obstacles that will work for me (please suggest other ideas if you have): 
- get a new delay on the same tab after I spent X minutes (for example, 10 min) in the tab
- automatically close a tab after I spent X minutes in the tab. This way if I want to continue browsing Twitter I have to open a new tab with it and get a delay. 

How do I accomplish that (or some other solution to my problem?) with Leechblock (or maybe other software if you know it)? 

My current solution is that I have enabled 24/7 delay when accessing Twitter feed. But it works only when I open a new Twitter tab. After I completed this delay I am completely free from any limits. I can't figure how to trigger new delays after some time on page or some other obstacles.

**Comment by proginosko** [comment, marked as answer]
Use two block sets for twitter.com. Set up the first one so that Twitter is blocked after X minutes of use. Set up the second one so that Twitter is always (24/7) blocked with the delaying page. They will work together so that Twitter is always delayed, and then eventually blocked after X minutes of use.
